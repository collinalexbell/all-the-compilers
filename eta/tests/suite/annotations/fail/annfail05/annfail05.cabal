name: annfail05
version: 1.0
build-type: Simple
cabal-version: >= 1.2

library
    build-depends: base
    exposed-modules: Annfail05
                     Annfail05_Help
