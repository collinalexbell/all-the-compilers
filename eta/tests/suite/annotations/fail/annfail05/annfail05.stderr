
[;1mAnnfail05.hs:11:1: [;1m[31merror:[0m[0m[;1m[0m[0m[;1m
    No instance for (Data.Data.Data NoInstances)
      arising from an annotation
    In the annotation: {-# ANN f NoInstances #-}[0m[0m
[;1m[38;5;31m   |[0m[0m
[;1m[38;5;31m11 |[0m[0m [;1m[31m{-# ANN f NoInstances #-}[0m[0m
[;1m[38;5;31m   |[0m[0m[;1m[31m ^^^^^^^^^^^^^^^^^^^^^^^^^[0m[0m
